fintech-examples/fintech-ui/src/app/bank-search/common/search/search.component.scss

```diff
@@ -17,6 +19,6 @@
     padding-top: 18px;
```

Comment:
why not to use rem or em instead of pixels ?

Reply:
for the search I want the unit to be fix.  We don't need a relative unit there